### PR TITLE
ci:Add envtest

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
Signed-off-by: Hyeongju Johannes Lee <hyeongju.lee@intel.com>

Closes: #566

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -51,6 +52,12 @@ jobs:
       - run: make go-mod-tidy
       - run: make BUILDTAGS=kerneldrv
       - run: make test BUILDTAGS=kerneldrv
+      - name: Install envtest tool and run envtest
+        run: |
+          go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          setup-envtest use ${K8S_VERSION}
+          source <(setup-envtest use -p env ${K8S_VERSION})
+          make envtest
```
Comment (mythi) [inline .github/workflows/ci.yaml:60]: not a big deal but I was thinking to keep install and run separate, i.e. have `- name: Install..` a bit earlier and then just:

```suggestion
      -run: make envtest
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ on:
 env:
   RUNC_VERSION: v1.0.2
   GO_VERSION: 1.17.2
+  K8S_VERSION: 1.20.2
```
Comment (mythi) [inline .github/workflows/ci.yaml:14]: let's find out why the latest does not work and add a comment here

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (hj-johannes-lee) [reply .github/workflows/ci.yaml]: Umm, I changed as you suggested, but there is ci error, I guess the last two lines have to be together.!

**Review by mythi — APPROVED** [review]

**Review by rojkov — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (hj-johannes-lee) [reply .github/workflows/ci.yaml:14]: #760

**Pull request merged**